.envrc

```
@@ -1,5 +1,8 @@
+# Use the default devShell from the flake.nix
+use flake .
+
 # Also load things from .env for developer-specific environment variables.
 dotenv
 
-# Use the default devShell from the flake.nix
-use flake .
+export HOST_PROJECT_PATH=$(pwd)
+export COMPOSE_PROJECT_NAME=galoy-quickstart
```

try to change to blink
```suggestion
export COMPOSE_PROJECT_NAME=blink-quickstart
```

---

Done